app/src/Api.tsx

```diff
@@ -158,6 +160,7 @@ export class FirebaseService {
 
       try {
         const rafflesEntered = (await get(ref(this.db, `users/${userId}/raffles_entered`))).val();
```

Comment:
I think this will solve your issue, the line below with the reduce is throwing an error when `rafflesEntered` is null, but will return `0` for `{}`

```suggestion
        const rafflesEntered = (await get(ref(this.db, `users/${userId}/raffles_entered`))).val() ?? {};
```